tests/installation/upgrade/select_upgrade_target_sle.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
```

Comment:
`this could be select_partitiong_or_system_to_update.pm`
We can take a look later, before adding `_sle` or something like that, for another ticket.

Reply:
It's because the module it replaces is also used in opensuse, so unless we use some logic or external test data, we'll probably want that soon.

Reply:
Therefore, I don't see why to create a ticket later for what we will most likely do later. I keep it for now, please confirm if you really want that, then I'll remove the _sle.

Reply:
My recommendation is to use the text on the screen everywhere, name of classes, name of modules, etc instead of introducing our own wording. so `select_partitiong_or_system_to_update.pm` is the most suitable name.
There is extensive vocabulary here if we want to do better (but still will not match with the screen, having more trouble for maintenance): https://documentation.suse.com/sles/15-SP4/single-html/SLES-upgrade/index.html
In the vocabulary there is migration target which is the system to be migrated to, but in this case in the name is confusing because we are selecting to system to migrate from, we should use Zero imagination/creativity for names and follow the screens given our own multi-language/culture squad characteristics :) my two cents.

Reply:
Ok, so in this case would we prefer some logic in the module based on variables, or some test data in the yaml schedule ? I thought this was the current approach, are we going back to test data ?

Reply:
Or, the specific string is in the page, and we create a different page for opensuse ?

Reply:
Why to tackle later when we can do it right away? I don't think we should never re-use code for Sle in Leap, they're 99% the same thing nowadays, so soon or late, we'll want to do something like this. With this particular example in mind, what would you see as the best way to select a different string in the same module ?

1- with test data ?
2- with logic based on variable (eg "if is_sle") ?
3- using Distribution pattern and a specific page (eg YAm/sle/SelectgForUpgradePage.pm) ?
4- by appending to the name "_sle" at the end of the module's name and using schedule ?
5- by just pressing next and not asserting what we select ?

Or something else ? We are changing the way we handle product variations regularly. In the end we have quite some mess, and the doc is obsoleted each time. Maybe we should decide something and stick to it once and for all, so we could document it and stop having the same discussion every 6 months ;-).

I fear you may choose option 5, but I like to actually assert what we select when possible.